common/dkg/src/bte/mod.rs

```diff
@@ -16,13 +15,11 @@ pub mod proof_sharing;
 pub use encryption::{decrypt_share, encrypt_shares, Ciphertexts};
 pub use keys::{keygen, DecryptionKey, PublicKey, PublicKeyWithProof};
 
-lazy_static! {
-    pub(crate) static ref PAIRING_BASE: Gt =
-        bls12_381::pairing(&G1Affine::generator(), &G2Affine::generator());
-    pub(crate) static ref G2_GENERATOR_PREPARED: G2Prepared =
-        G2Prepared::from(G2Affine::generator());
-    pub(crate) static ref DEFAULT_BSGS_TABLE: encryption::BabyStepGiantStepLookup =
-        encryption::BabyStepGiantStepLookup::default();
+pub(crate) fn precompute_pairing_base() -> Gt {
```

Comment:
this will cause the value to be recomputed every time it is called (which is relatively expensive). `lazy_static` was preventing that. however, because rust has evolved since this has been written, you can replace it with inbuilt static`LazyCell` (or `LazyLock`)

Reply:
Ok

Reply:
it wasn't about the size but the computation time : )